jsonnet/openebs-mixin/config.libsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  // openebs-mixin configurations
+  _config+:: {
+    // dashboards configuration. If set, then dashboards json will be generated.
+    dashboards: {
+      cStor: true,
+      Jiva: true,
+      LocalPV: true,
+    },
+    // AlertRules configuration. If set, then rules json will be generated.
+    alertRules: {
+      cStor: true,
+      Jiva: true,
+      lvmLocalPV: false,
+      volume: true,
+      npd: true,
+    },
+  },
+}
```

Comment:
Can we make everything in lower-case to avoid any confusion?

Reply:
We can but it might create confusion for lvmlocalpv or devicelocalpv.

Reply:
I don't think there will be any confusion.

Reply:
Okay. Will update it.